docs/tools-resources/resources.mdx

```diff
@@ -93,6 +93,7 @@ Disclaimer: These tools and guides are created by the community and are not supp
 * [drep.tools](https://drep.tools/): A very early version of a site to help DReps.
 * [Plutarch](https://github.com/Plutonomicon/plutarch-plutus): A Haskell eDSL designed for writing fast and fine-tunable on-chain scripts. Currently supports PlutusLedger V1 and V2, with [V3 on its way](https://github.com/Plutonomicon/plutarch-plutus/issues/655).
 * [carp 3.2.0](https://github.com/dcSpark/carp/releases/tag/3.2.0) This project is designed to sync data from the Cardano blockchain and store it into a Postgres database.
+* [Constitution Checker](http://ec2-16-171-11-232.eu-north-1.compute.amazonaws.com:8080/) Constitution checker is a web-based tool that checks the constitutionality of a Parameter Update Proposal. It implements checks for the guardrails from the [Constitution](https://raw.githubusercontent.com/IntersectMBO/interim-constitution/main/cardano-constitution-0.txt).
```

Comment:
```suggestion
* [Parameter Change Guardrails Checker](http://ec2-16-171-11-232.eu-north-1.compute.amazonaws.com:8080/) Parameter Change Guardrails Checker]is a web-based tool that checks the constitutionality of a Parameter Update Proposal. It implements checks for the guardrails from the [Constitution](https://raw.githubusercontent.com/IntersectMBO/interim-constitution/main/cardano-constitution-0.txt).
```

I feel the name constitution checker is quite ambiguous and may lead to misinterpretations
Id suggest a change to something more specific

Reply:
What about Parameter Proposal Constitutionality Checker?

Reply:
How about `Parameter Proposal Constitutionality Checker`

Reply:
`Parameter Proposal Constitutionality Checker`

I prefer using the term guard rails as that is what the tool is doing
I think constitutionality is too subjective as an action could pass the guardrails (and the tool) but is then deemed unconstitutional by the committee

Reply:
Are you proposing to change the name to 'Parameter Proposal Guardrail Checker' or just 'Guard Rails'?

Reply:
Hey @Ryun1 just following up on this. Do you have a preference for either one of the names listed above?

Reply:
Parameter Proposal Guardrails Checker?

Reply:
@Ryun1 yes, I think that will work great! I just pushed a commit with the updated name.

Reply:
awesome